### ISSUE TITLE
Transfer learning : accuracy decreases in eval mode

### ISSUE THREAD

**Issue by jplumail** [princeton-vl/pytorch_stacked_hourglass#34]
Hi, thanks for maintaining this project !

I had some problem when doing transfer learning with the pretrained 2 Hourglass model on another dataset (not related to human pose problem but quite close).
I experienced some problems in eval mode on my validation and my training set. When I set `track_running_mean=False` in Batchnorm layers, everything works fine. I tried to change the momentum, change the batch size but it doesn't solve the problem.

Have you ever experienced something like that with Stacked Hourglass ?

Thanks !

**Comment by crockwell** [comment]
Hmm... Batchnorm can be tricky. I haven't personally run into this error on Hourglass so unfortunately I can't help you much. Because normalization performs differently on train vs. test, I can imagine transfer learning being especially difficult... One thing I can think of is to be extra careful if beginning from a pretrained model! Sorry I can't be of more help.
